Edge10.CouchDb.Client/ICouchModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Edge10.CouchDb.Client
+{
+	/// <summary>
+	/// An interface for model classes stored in CouchDB.
+	/// </summary>
+	public interface ICouchModel
+	{
+		/// <summary>
+		/// The CouchDB ID for this document.  This will be converted to a <see cref="string"/>
+		/// for DB storage.
+		/// </summary>
+		Guid Id { get; set; }
```

Comment:
Should we force everything to use a `Guid` as an ID?  Couch supports any `string` so if this is going to be a generic library I feel like it would be to use strings.

We also have some `ICouchApi` methods that accept IDs as strings!